WebDriverAgentLib/Utilities/FBMjpegServer.m

```diff
@@ -82,16 +82,14 @@ - (void)streamScreenshot
   CGFloat compressionQuality = FBConfiguration.mjpegServerScreenshotQuality / 100.0f;
   id<XCTestManager_ManagerInterface> proxy = [FBXCTestDaemonsProxy testRunnerProxy];
   dispatch_semaphore_t sem = dispatch_semaphore_create(0);
-  [proxy _XCT_setAXTimeout:SCREENSHOT_TIMEOUT reply:^(int res) {
```

Comment:
should we also remove the unused SCREENSHOT_TIMEOUT constant?

Reply:
it is still used in the semaphore timeout, right?

Reply:
yep, makes sense